index.css

```diff
@@ -0,0 +1,94 @@
+@import url(../stylesheets/layout/navbar.css);
+@import url(../stylesheets/base/reset.css);
+@import url(../stylesheets/base/root-variables.css);
+
+main{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding-top: 5rem;
+    gap: 1rem;
+}
+
+main .head{
+    margin: auto;
+    max-width: 700px;
+    width: 80%;
+    text-align: center;
+    color: black;
+    font-size: 3.1rem;
+    font-weight: 900;
+}
+
+main .sub-head{
+    margin: auto;
+    max-width: 750px;
+    width: 90%;
+    text-align: center;
+    font-weight: 400;
+    font-size: 1.3rem;
+    color: var(--light-text-color);
+}
+
+.btn-section{
+    display: flex;
+    justify-content: space-evenly;
+    width: 20rem;
+    margin: auto;
+    gap: 1rem;
+}
+
+.btn{
+    min-width: 7.1rem;
+    font-size: 1.2rem;
+    font-weight: 600;
+    border-radius: 5px;
+    padding: 0.7rem;
+    margin-top: 1rem;
+    box-shadow: var(--box-shadow-btn);
+}
+
+ .btn-primary{
+    background-color: var(--primary-color);
+    color: white;
+    text-align: center;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color);
+    color : var(--primary-color);
+    border: 1px solid var(--primary-color);
+    text-align: center;
+}
```

Comment:
Can move this `text-align:center` to `.btn` class

Reply:
ok thanks, will do!